README.md

```diff
@@ -164,3 +168,21 @@ go run main.go integration-tests
 - Both `db` and `stellar-rpc` services should be running, either in Docker containers or locally.
 
 This setup allows you to verify both the isolated functionality of components (unit tests) and their interactions (integration tests) within the wallet-backend.
+
+## Docker Hub Publishing
+
+The CI/CD workflow, defined in [`publish_to_docker_hub.yaml`](./.github/workflows/publish_to_docker_hub.yaml) automates the process of building and publishing Docker images to Docker Hub. This workflow is triggered under two conditions:
+
+### Push to `develop`
+
+Whenever updates are made to the `develop` branch, the workflow creates and uploads a Docker image with two specific tags:
+
+- `edge`
```

Comment:
is this equivalent to the `testing` tag?

Reply:
Yes. I ported this from the SDP. At the time we did some research and found that `edge`/`dev`/`testing` were the most common wordings used for this. We ended up using `edge` there, but we could adopt another word in the wallet-backend.

Reply:
We decided to use `testing`

Reply:
https://github.com/stellar/wallet-backend/pull/169/commits/dbada52ed849b31cb1c30b081ac9638a58904fef